### PR TITLE
Data migration to replace policy admin URLs in content.

### PR DESCRIPTION
**Pull request by elliotcm**
These will stop working once the code and/or data is removed, and so need replacing with their full public URL.

Output when run locally: https://gist.github.com/elliotcm/3c63f101d6d49241a912

https://trello.com/c/8nYU9WTp/324-replace-admin-links-for-policies-and-supporting-pages-with-static-public-links

### CODE REVIEW

File: app/helpers/public_document_routes_helper.rb [hunk 1]
```diff
@@ -93,9 +97,15 @@ def build_url_for_corporate_information_page(edition, options)
     end
   end
 
-  def build_url_for_supporting_page(edition, options)
+  def build_url_for_supporting_page(edition, options, builder_options)
     options = options.merge(id: edition.document)
-    options[:policy_id] ||= edition.related_policies.first.document
+
+    options[:policy_id] ||= if builder_options[:include_deleted_documents]
```
Comment (edds) [inline app/helpers/public_document_routes_helper.rb]: I am finding this a touch hard to parse. Is this equivalent to:

```ruby
if builder_options[:include_deleted_documents]
  options[:policy_id] ||= edition.related_policies.unscoped.first.document
else
  options[:policy_id] ||= edition.related_policies.first.document
end
```

Comment (boffbowsh) [inline app/helpers/public_document_routes_helper.rb]: It's essentially Ternary But For Long Lines.
```ruby
options[:policy_id] ||= builder_options[:include_deleted_documents] ? edition.related_policies.unscoped.first.document : edition.related_policies.first.document
```

Comment (elliotcm) [inline app/helpers/public_document_routes_helper.rb]: It also contains a bug I'm working on.  But yes it is.

**Comment by elliotcm** [comment]
@boffbowsh @edds I've updated the tests and code to better handle deleted policies/supporting pages.  This should hopefully also be an easier to parse conditional.

**Comment by edds** [comment]
:+1:
